### PR TITLE
Add coordgenlibs

### PR DESCRIPTION
**Pull request by mojaie**
Package name: coordgenlibs
https://github.com/schrodinger/coordgenlibs

### CODE REVIEW

File: C/coordgenlibs/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "coordgenlibs"
+version = v"0.0.1"
```
Comment (giordano) [inline C/coordgenlibs/build_tarballs.jl]: Could you please use the corresponding upstream version number?

**Comment by mojaie** [comment]
v1.3.2 have Boost dependency which makes the automatic build difficult. The dependency was removed from the master branch recently. Should I wait for new release?

**Comment by giordano** [comment]
Ok, let's pretend this is 1.3.2 on steroids and call it simply v1.3.2

File: C/coordgenlibs/build_tarballs.jl [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+script = raw"""
+cd $WORKSPACE/srcdir/coordgenlibs
+cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release -DCOORDGEN_BUILD_TESTS=OFF -DCOORDGEN_BUILD_EXAMPLE=OFF
+make -j${nproc}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
```
Comment (giordano) [inline C/coordgenlibs/build_tarballs.jl]: The audit shows the following warning:
```
┌ Warning: /agent/_work/1/s/C/coordgenlibs/build/i686-linux-gnu/HYvPpuwG/destdir/lib/libcoordgen.so.1.3.2 contains std::string values!  This causes incompatibilities across the GCC 4/5 version boundary.  To remedy this, you must build a tarball for both GCC 4 and GCC 5.  To do this, immediately after your `platforms` definition in your `build_tarballs.jl` file, add the line:
│ 
│     platforms = expand_cxxstring_abis(platforms)
└ @ BinaryBuilder /depot/packages/BinaryBuilder/4fMlX/src/auditor/compiler_abi.jl:189
```
Need to modify
```suggestion
platforms = expand_cxxstring_abis(supported_platforms())
```

**Comment by mojaie** [comment]
Thank you very much!

**Comment by giordano** [comment]
Thanks to you for the contribution! :wink:

**Pull request merged**